modules/ROOT/pages/6.8.1-release-notes.adoc

```diff
@@ -541,6 +541,16 @@ As a result, these modifications significantly enhance the accessibility of auto
 // #TINY-10141
 In {productname} 6.7.0, each cell in the table grid menu item was updated so that it would announce the number of columns and rows currently selected.
 
-In {productname} 6.8.1.0 translations were added for this text, though this announcement was still only available in English as the syntax used for the table grid announcement text in detecting strings which are to be translated was invalid.
+In {productname} 6.8.1 translations were added for this text, though this announcement was still only available in English as the syntax used for the table grid announcement text in detecting strings which are to be translated was invalid.
 
-{productname} 6.8.1.1 addresses this, now, the syntax has been corrected and each cell in the table grid announces the current selected columns and rows in the correct language, _not only_ in English.
\ No newline at end of file
+{productname} 6.8.1 addresses this, now, the syntax has been corrected and each cell in the table grid announces the current selected columns and rows in the correct language, _not only_ in English.
+
+=== The functions schema.isWrapper and schema.isInline didn't exclude element names that started with # those should not be considered elements.
+// #TINY-10385
+Previously the `isInline` and `isWrapper` functions in the Schema module would incorrectly output `true` for non-element nodes such as `#text` and `#comment`.
+
+As a consequence, in certain cases the editor would attempt to append a child node inside such a non-element node, which is forbidden and results in an error being thrown.
+
+In {productname} 6.8.1, `Schema.isInline` and `Schema.isWrapper` exclude non-element nodes whose names starts with `#`, such as `#text` and `#comment`.
+
+As a result, these elements are no longer considered as valid inline or wrapper elements, preventing errors caused by element methods being called on non-element nodes.
```

Comment:
I'm not sure that `non-element` is correct, nodes that has name that start with `#` are not eligible as `wrapper` element but I'm not sure that the element definition doesn't fit them